test/fixtures/cookbooks/rack_user_test/recipes/remove.rb

```diff
@@ -1,5 +1,10 @@
 # comments!
 
+user 'rack' do
```

Comment:
Should we be using 'rack_user' here instead of user? So we're testing that we can remove the rack user we added, vs. testing we can remove a user added by a different provider?